packages/backend/src/libs/classes/RESTController.ts

```diff
@@ -23,10 +24,10 @@ export class RESTController<T> {
     }
   }
 
-  public store(body: T): Promise<T> {
+  public async store(body: T): Promise<T> {
     try {
       const entity = this.repository.create(body);
-      return this.repository.save(entity);
+      return await this.repository.save(entity);
```

Comment:
the code ```return await``` is redundant. See https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md

Reply:
thanks, interesting